site/src/scaffold/code-block.scss

```diff
@@ -39,5 +43,10 @@ $code-block-header-font-size: 0.8em !default;
 
 	.code-block-body {
 		@include code-block();
+
+		&.code-bock-body-max-height {
```

Comment:
should these two code-block specific classes be move to just some height related classes?

```
.overflow-x-scroll (this one looks like it could be included in the framework, but we can hold off on that for now)
.max-height-30vh
.min-height-30vh

```

Reply:
I could see that making a lot of sense. I think we landed on the code-block.scss because the inline css we were replacing was only being placed on the code-blocks elements.

Reply:
Let's create a new scss file in `site\src\scaffold\styles\` folder (call it `misc.scss`?) and add these three new classes into it. Just don't firget to import it in `site\src\scaffold\index.scss` as well ;)